gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagManager.java

```diff
@@ -306,6 +306,8 @@ protected void startUp() {
    * Note this should only be called from the {@link Orchestrator} or {@link org.apache.gobblin.service.monitoring.DagActionStoreChangeMonitor}
    */
   public synchronized void addDag(Dag<JobExecutionPlan> dag, boolean persist, boolean setStatus) throws IOException {
+    // TODO: Additional log added here to track missing dag issue, remove later as needed
+    log.info("Add dag called for dag: {} to be persisted: {} and status set: {}", dag, persist, setStatus);
```

Comment:
nit:
```
log.info("addDag(persist: {}; setStatus: {}): {}", persist, setStatus, dag)
```
(to give the short and easily parseable log msgs I most prefer working with)